extensions/wikia/Forum/RelatedForumDiscussionController.class.php

```diff
@@ -13,7 +13,7 @@ public function init() {
 
 	public function index() {
 
-		$this->messages = $this->getVal( 'messages' );
+		$this->response->setVal('messages', $this->getVal( 'messages' ));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "messages" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)